lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -375,6 +377,38 @@ public void setAnimationFromUrl(String url) {
     setCompositionTask(LottieCompositionFactory.fromUrl(getContext(), url));
   }
 
+  /**
+   * Sets a task listener. Previous listener will be removed
+   *
+   * @param loadedListener - task listener
+   * @see LottieTask#addListener(LottieListener)
+   * @see LottieTask#removeListener(LottieListener)
+   */
+  public void setLoadedListener(@NonNull LottieListener<LottieComposition> loadedListener) {
```

Comment:
`add`/`remove` would be better than `set`/`remove` (so you need to keep track of a list)

If the task changes after you set the listener, it also won't add your listener

Reply:
@iismagilov